src/index.js

```diff
@@ -0,0 +1,27 @@
+import './style.css';
+import {
+  loadTasks,
+  addTask,
+  deleteTaks,
+  updateTaks,
+  deleteAllTaks,
+} from './modules/functions.js';
+import changeStatus from './modules/statusUpdate.js';
+
+const ul = document.querySelector('.ul');
+loadTasks(localStorage.getItem('lists'), ul);
+const addInput = document.querySelector('.add-input form');
```

Comment:
This is a suggestion buddy: Avoid the use of  *querySelector*, use more specific selectors like *getElementByID* or *getElementByClassName*. It makes your code more readable and easier to maintain 🚶